JS/company.js

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
this file has extra indentation for all lines. why?

Reply:
By mistake, Corrected.